setup.py

```diff
@@ -173,7 +173,7 @@ def _header_bad(self, path):
     geos_includes = []
     geos_library_dirs = []
     if sys.platform.startswith('win'):
-        geos_libraries = ['geos']
+        geos_libraries = ['geos_c', 'geos']
```

Comment:
Not sure if we need both.

Reply:
We shouldn't need both. https://github.com/Toblerity/Shapely/issues/343#issuecomment-157639838 suggests we can use geos_c. This is indeed what shapely do https://github.com/Toblerity/Shapely/pull/400/files.